elk-setting/docker-compose.yml

```
@@ -0,0 +1,208 @@
+      '
+    healthcheck:  # 서비스 헬스체크 설정
+      test: ["CMD-SHELL", "[ -f config/certs/es01/es01.crt ]"]  # es01 인증서 파일 존재 여부 확인
+      interval: 1s  # 헬스체크 간격 1초
+      timeout: 5s  # 헬스체크 타임아웃 5초
+      retries: 120  # 헬스체크 재시도 횟수 120회
+
+  es01:  # Elasticsearch 인스턴스 정의
+    depends_on:  # setup 서비스에 의존
+      setup:
+        condition: service_healthy  # setup 서비스가 헬스체크를 통과한 경우에만 시작
+#    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}  # Elasticsearch 이미지 사용
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)